proseco/tests/test_guide.py

```diff
@@ -296,7 +296,7 @@ def test_check_spoiler_cases():
             stars.add_fake_star(row=r, col=c, mag=mag0, id=1, ASPQ1=1)
             # Add a "spoiling" star and move it from center past edge through
             # the drcs
-            stars.add_fake_star(row=r + drc, col=c, mag=mag0 + dmag, id=2, ASPQ1=0)
+            stars.add_fake_star(row=r + drc, col=c, mag=mag0 + dmag, id=2, ASPQ1=30)
```

Comment:
Why are you changing this existing test?

Reply:
So that the id=2 star can't be used as a guide star, thus hitting our new check.

Reply:
Though probably should have changed the test by making the fake star un-selectable via class or some such that is not a stage-selection-influencing piece.